src/backend/marsha/settings.py

```diff
@@ -158,6 +158,9 @@ class Base(Configuration):
         "EXCEPTION_HANDLER": "marsha.core.views.exception_handler",
         "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.LimitOffsetPagination",
         "PAGE_SIZE": 50,
+        "DEFAULT_THROTTLE_RATES": {
+            "anon": "3/minute",
```

Comment:
So 3/minutes during 3 minutes is not 3 attempts like we discussed earlier but 9 attempts? In this case it should be set to 1/minute or expiry should be set to 1 minute?

Reply:
We still need to discuss this point.
Maybe looking at related tests would help us to find a good balance ?